doc/source/user/numpy-for-matlab-users.rst

```diff
@@ -258,63 +187,63 @@ Linear Algebra Equivalents
      - Notes
 
    * - ``ndims(a)``
-     - ``ndim(a)`` or ``a.ndim``
-     - get the number of dimensions of an array
+     - ``np.ndim(a)`` or ``a.ndim``
+     - get the number of dimensions of the array, ``a``
 
    * - ``numel(a)``
-     - ``size(a)`` or ``a.size``
-     - get the number of elements of an array
+     - ``np.size(a)`` or ``a.size``
+     - get the number of elements of the array, ``a``
 
    * - ``size(a)``
-     - ``shape(a)`` or ``a.shape``
-     - get the "size" of the matrix
+     - ``np.shape(a)`` or ``a.shape``
+     - get the "size" of the array, ``a``
 
    * - ``size(a,n)``
      - ``a.shape[n-1]``
      - get the number of elements of the n-th dimension of array ``a``. (Note
-       that MATLAB® uses 1 based indexing while Python uses 0 based indexing,
+       that MATLAB uses 1 based indexing while Python uses 0 based indexing,
        See note :ref:`INDEXING <numpy-for-matlab-users.notes>`)
 
    * - ``[ 1 2 3; 4 5 6 ]``
-     - ``array([[1.,2.,3.], [4.,5.,6.]])``
-     - 2x3 matrix literal
+     - ``np.array([[1.,2.,3.], [4.,5.,6.]])``
+     - 2D array defined as 2x3 matrix
 
    * - ``[ a b; c d ]``
-     - ``block([[a,b], [c,d]])``
+     - ``np.block([[a,b], [c,d]])``
      - construct a matrix from blocks ``a``, ``b``, ``c``, and ``d``
 
    * - ``a(end)``
      - ``a[-1]``
-     - access last element in the 1xn matrix ``a``
+     - access last element in MATLAB vector (1xn or nx1) or 1D NumPy array
+       ``a`` (length n)
 
    * - ``a(2,5)``
      - ``a[1,4]``
-     - access element in second row, fifth column
+     - access element in second row, fifth column in a 2D array, ``a``
 
    * - ``a(2,:)``
      - ``a[1]`` or  ``a[1,:]``
-     - entire second row of ``a``
+     - entire second row of a 2D array, ``a``
```

Comment:
```suggestion
     - entire second row of 2D array ``a``
```

Reply:
This isn't really gramatically reasonable any more, even as a fragment - we can drop the articles at the start of the description, but it feels wrong to do so mid-description.

Reply:
@eric-wieser  A grammatically parallel construction is  "Footprint of Apollo astronaut Neil Armstrong."

Reply:
It's apposition, is legal, and nobody thinks twice.  "Sister of advice columnist Ann Landers." "Cover of book by NumPy creator Travis Oliphant." 

Consider alternatives:

<br>

> Footprint of the Apollo astronaut Neil Armstrong

Only if we're distinguishing it from "Footprint of the Indiana lawyer Neil Armstrong."

<br>

> Footprint of the Apollo astronaut, Neil Armstrong

This says there was only one Apollo astronaut.

<br>

> Footprint of an Apollo astronaut, Neil Armstrong

This suggests we're contrasting "Footprint of a Gemini astronaut, Gus Grissom."

<br>

> Footprint of an Apollo astronaut Neil Armstrong

Bad usage.

<br>

> Footprint of Apollo astronaut, Neil Armstrong

Bad usage.

<br>

"Footprint of Apollo astronaut Neil Armstrong" is in fact the best way to put it in this context, as is "entire second row of 2D array ``a``."